esp/esp/utils/widgets.py

```diff
@@ -17,14 +17,9 @@
 
 # DATETIMEWIDGET
 calEnable = u"""
+<span id="{id}-info"><span id="{id}-gi"></span> <span id="{id}-duration"></span></span>
 <script type="text/javascript">
-    $j("#%s").%s({
-        showOn: 'button',
-        buttonImage: '%simages/calbutton_tight.png',
-        buttonImageOnly: true,
-        dateFormat: '%s',
-        timeFormat: '%s'
-    });
+    setupDatepickerDurationLabel('{widget}', '{id}', '{media_url}', '{date_format}', '{time_format}');
 </script>"""
 
 class DateTimeWidget(forms.widgets.TextInput):
```

Comment:
These widgets appear in a bunch of places. I don't think there's anywhere where it's bad to have this display (if such a place would exist), but some pages (e.g. volunteering, accessible via manage -> volunteers) make it wrap weirdly.

Reply:
OK, I made the textboxes narrower so wrapping is less likely. It might not fix this in all cases, but should be good enough in most.